public/js/safetymaps/safetymaps.js

```diff
@@ -206,7 +204,7 @@ safetymaps.infoWindow = {
             }
             tabId = s[1];
             showWindow = !!showWindow;
-            console.log("showTab shorthand windowId=" + windowId + ",tabId=" + tabId + ",showWindow=" + showWindow);
+            //console.log("showTab shorthand windowId=" + windowId + ",tabId=" + tabId + ",showWindow=" + showWindow);
```

Comment:
Niet helemaal verwijderen?

Reply:
Zou eigenlijk ook achter setting moeten, maar zit niet in een module dus is nu even geen plek voor. Dus als deze log regel in de toekomst nuttig is staat ie iig nu zo.